src/index.html

```diff
@@ -226,35 +226,7 @@ <h3 class="sr-only">Clean up and add new step</h3>
     </div>
 
     <!-- BEGIN FOOTER -->
-    <div id="footer" class="footer">
-        <footer class="footer"
-                style="margin-bottom: 10px; background-color: #001F3F; border-top: 10px solid #001F3F; display: flex; border-bottom: 10px solid #001F3F;">
-            <h2 class="sr-only"> Footer </h2>
-            <div style="flex: 0 0 100px;">
-                <a href="https://www2.ed.gov/about/offices/list/osers/osep/index.html" target="_blank">
-                    <img style="float: left; margin: 0px 25px 0px 10px; background-color: #FFF; padding: 5px;"
-                         src="images/IDEA-logo.gif" alt="IDEA Logo" height="50">
-                </a>
-            </div>
-            <div style="font-size: 0.7em; color: #FFF;">
-                The <a href="http://diagramcenter.org/" target="_blank">DIAGRAM Center</a> is a <a
-                    href="http://www.benetech.org/"
-                    target="_blank">Benetech</a>
-                initiative supported by the U.S. Department of Education, Office of Special Education Programs (Cooperative
-                Agreement
-                #H327B100001). Opinions expressed herein are those of the authors and do not necessarily represent the
-                position of
-                the U.S. Department of Education. Poet™ is a trademark of Beneficent Technology, Inc. This website is
-                copyright ©
-                2012-2017, Beneficent Technology, Inc.
-            </div>
-            <div style="flex: 0 0 100px;">
-                <a href="https://www.benetech.org/" target="_blank">
-                    <img style="float: right;margin: 0px 10px 0px 25px; background-color: #FFF; padding: 5px;"
-                         src="images/benetech_logo_transparent.gif" alt="Benetech Logo" height="50">
-                </a>
-            </div>
-        </footer>
+    <div id="footerTestPlaceHolder">
```

Comment:
Why not just "footer"?

Reply:
I think it is because it is the temporary solution. Maybe footerTestPlaceHolder would be better...

Reply:
footerPlaceHolder sounds good as well for me

Reply:
exactly, fine then, I've removed "test"